docker/compose/nginx/files/nginx.conf

```diff
@@ -9,6 +9,7 @@ events {
 http {
   include /etc/nginx/mime.types;
   default_type application/octet-stream;
+  resolver 127.0.0.11;
```

Comment:
This won't be consistent every time, the only reason we have resolver specified in prod is for auth0 integration, so I don't think we need this line.

Reply:
This is a docker embedded construct for  the nginx to be able to query DNS for the other containers, it's always this ip.  This has no relation to the resolver we have in the production nginx config. We add this because we have our networks defined.

https://docs.docker.com/v17.09/engine/userguide/networking/configure-dns/